clearnode/docs/Clearnode.protocol.md

```diff
@@ -1,18 +1,21 @@
 # Clearnode Protocol Specification
```

Comment:
@philanton , actually, yes I would like to merge it after implementing changes to the node and sdk. Also, a question to you as a person that worked on RPCNode, what do you think about using object for params instead of an array (currently I'm fixing `rpc_node_test` and it's quite painful)